cluster/images/etcd/Makefile

```
@@ -83,7 +83,7 @@ endif
 # This option is for running docker manifest command
 export DOCKER_CLI_EXPERIMENTAL := enabled
 # golang version should match the golang version of the official build from https://github.com/etcd-io/etcd/releases.
-GOLANG_VERSION := 1.20.10
+GOLANG_VERSION := 1.20.11
```

Just before the release we updated golang to 1.20.12 https://github.com/etcd-io/etcd/blob/v3.5.11/.go-version
```suggestion
GOLANG_VERSION := 1.20.12
```

---

Done